packages/kbn-openapi-generator/src/parser/lib/helpers/find_refs.ts

```diff
@@ -6,8 +6,8 @@
  * Side Public License, v 1.
  */
 
-import { hasRef } from './helpers/has_ref';
-import { traverseObject } from './helpers/traverse_object';
+import { hasRef } from './has_ref';
+import { traverseObject } from './traverse_object';
 
 /**
  * Traverse the OpenAPI document recursively and find all references
```

Comment:
Technically it's possible but it's not convenient. There are cases when this function should accept an OpenAPI Document subpart like `components` object or some schema object. Specifying all that possibilities is cumbersome and doesn't give any benefits but increases maintenance burden. For example `openapi-types` package exposes types for OpenAPI specification `3.0` as `OpenAPIV3` and for `3.1` as `OpenAPIV3_1`. When we decide to migrate to OpenAPI specification `3.1` we'll need to replace `OpenAPIV3` usage with `OpenAPIV3_1`. It can be simplified by exporting abstracted types from `packages/kbn-openapi-generator/src/parser/openapi_types.ts` but it will still require some work. `unknown` won't require any changes. `traverseObject` helper function used under the hood performs necessary checks.